CodeReview/src/CodeReviewManager.cpp

```diff
@@ -88,6 +89,8 @@ void CodeReviewManager::saveReview(QString newVersion)
 NodeReviewsList* CodeReviewManager::loadReview(QString newVersion, VersionControlUI::DiffSetup& diffSetup,
 																 Visualization::ViewItem* viewItem)
 {
+	CFocus::clearFocusInformation();
```

Comment:
I like how the rest is nicely separated, just that you call here clear/extract focus.

Do we need this here? Can't we do both of these only when the user invokes the `CFocus` command?  Or what do you have in mind?